crates/virtio-queue/CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # Upcoming
 
+# v0.10.0
+
+Identical to v0.9.1, which was incorrectly published as minor release.
+
 # v0.9.1
 
+**yanked**: This version got yanked. It should have been a major release. The
```

Comment:
This is a great idea! Maybe we can use v0.9.1 - yanked in the header title.

Reply:
Good idea. Will move the `**yanked** to the header.